gtfu/src/main/java/gtfu/tools/TripListGenerator.java

```diff
@@ -300,19 +309,30 @@ public static void main(String[] arg) throws Exception {
             out.close();
         }
     }
+
+    // For agencies who don't include the optional directions.txt file, assume the following mapping.
+    // This is designed specifically around Unitrans
```

Comment:
My recollection from reading the spec is that the meaning  of direction symbols is completely arbitrary. Please confirm. A better solution may be to make Unitrans add directions.txt

Reply:
Yes that's correct - definitely better for them to add directions.txt and I'll find a way to make that happen.